src/gameui.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-alert */
```

Comment:
**Can all inline eslint rule be removed, please define eslint rule in the eslint file.**

Reply:
Yes. Thanks for pointing out